pkg/preflight/assets/host-preflights.yaml

```diff
@@ -360,3 +363,19 @@ spec:
           - fail:
               when: "p99 >= 20ms"
               message: "Write latency is too high. p99 target < 10ms, actual:{{ .String }}"
+    - diskUsage:
+        checkName: "Longhorn Disk Usage"
+        collectorName: "Longhorn Disk Usage"
+        exclude: '{{kurl and .Installer.Spec.Longhorn.Version (not .IsUpgrade) | not }}' # only run if there is a longhorn version set and this is not an upgrade
+        outcomes:
+        - fail:
+            when: "total < 50Gi"
+            message: The disk containing directory /var/lib/longhorn has less than 50Gi of total space
+        - fail:
+            when: "used/total > 80%"
+            message: The disk containing directory /var/lib/longhorn is more than 80% full
+        - warn:
+            when: "used/total > 60%"
+            message: The disk containing directory /var/lib/longhorn is more than 60% full
+        - pass:
+            message: The disk containing directory /var/lib/longhorn has at least 30Gi disk space available
```

Comment:
```suggestion
            message: The disk containing directory /var/lib/longhorn has at least 20Gi disk space available
```

Reply:
"The disk containing directory /var/lib/longhorn has at least 20Gi disk space available and is at least 50Gi" would be even better, will do